generators/generator-base-private.js

```diff
@@ -795,55 +795,47 @@ module.exports = class extends Generator {
      * @return {string} version - retrieved version or empty string if not found
      */
     findBlueprintVersion(blueprintPkgName) {
-        let packageJsonPath = path.join(process.cwd(), 'node_modules', blueprintPkgName, 'package.json');
-        try {
-            if (!fs.existsSync(packageJsonPath)) {
-                this.debug('using global module as local version could not be found in node_modules');
-                packageJsonPath = path.join(blueprintPkgName, 'package.json');
+        const blueprintGeneratorName = blueprintPkgName.replace('generator-', '');
```

Comment:
This specific replacement is done multiple times, may be lets have a util method as below to do it
```
const normalizeGeneratorName = name => name.replace('generator-', '');
```

Reply:
Yeah, thanks. 
It belongs to the things I need to finish (with fixing upgrade unit test, merging app-blueprint.spec.js and scoped-blueprint.spec.js and fixing conflicts with https://github.com/jhipster/generator-jhipster/pull/10541), that's why it's still a draft. 

But I opened it in order to get some feedback, to know if I should continue 😉